### DISCUSSION TITLE
Python Transform to count frams

### DISCUSSION THREAD

**Discussion by ManalKR** [bonsai-rx/bonsai#1667] [Q&A]
Hi, 

I am trying to count the video frams from by using python transform node, but something is wrong and not sure what. Why it keeps giving me an error?
![Screenshot 2024-01-29 162159](https://github.com/bonsai-rx/bonsai/assets/127253474/3c212ad9-6729-40b5-9192-78529e55916a)

**Comment by bruno-f-cruz** [comment]
You defined the function body without indenting.
